packages/dashboard/src/vaadin-dashboard.js

```diff
@@ -374,14 +366,20 @@ class Dashboard extends DashboardLayoutMixin(ElementMixin(ThemableMixin(PolylitM
       return wrapper;
     }
 
+    // Sibling wrappers sorted by their slot name
+    const siblingWrappers = [...wrapper.parentElement.children].sort((a, b) => {
+      return a.slot < b.slot ? -1 : 1;
```

Comment:
We need something more sophisticated here: `'slot-11' < 'slot-2' === true`

Reply:
Good point, will update the commit to `flow-components` aslo